src/styles/navigation_item.css

```diff
@@ -23,8 +23,8 @@
 }
 
 /* Hover effect */
-[role=banner] .coz-nav-item a[role=menuitem]:hover:before,
-[role=banner] .coz-nav-item a[role=menuitem]:focus:before,
+[role=banner] .coz-nav-item a[role=menuitem]:not(.coz-bar-coming-soon-app):hover:before,
```

Comment:
This selector is getting more insane every time… Wouldn't be simpler to just… i don't know, not putting the coming soon item inside a `<a>` tag?